frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "react-table": "^7.8.0",
     "resolve": "^1.20.0",
     "semver": "^7.3.5",
+    "swagger-ui-react": "^5.10.3",
```

Comment:
Should we limit Swagger UI and FastAPI docs to just the development environment?

Reply:
done. thanks for the suggestion